src/ocean/util/app/ext/LogExt.d

```diff
@@ -152,18 +152,27 @@ class LogExt : IConfigExtExtension
         auto log_config = ConfigFiller.iterate!(LogUtil.Config)("LOG", config);
         auto log_meta_config = ConfigFiller.fill!(LogUtil.MetaConfig)("LOG", config);
 
+        enable_loose_parsing(conf_ext.loose_config_parsing);
+
+        // tracks already registered files to prevent double registration
+        // by both old and new logger system. Can be removed in ocean v4
+        File[cstring] registered_files;
```

Comment:
Please add `deprecated` somewhere in the comment, so we can grep for it when removing deprecations ;-)